event/src/main/java/com/bazaarvoice/emodb/event/db/astyanax/AstyanaxEventWriterDAO.java

```diff
@@ -43,6 +46,7 @@ public AstyanaxEventWriterDAO(CassandraKeyspace keyspace, SlabAllocator slabAllo
         _channelConfiguration = channelConfiguration;
     }
 
+
     @ParameterizedTimed(type="AstyanaxEventWriterDAO")
     @Override
     public void addAll(Multimap<String, ByteBuffer> eventsByChannel, @Nullable EventSink sink) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 18 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS3776)